nautobot/extras/tests/example_jobs/test_job_hook_receiver.py

```diff
@@ -1,13 +1,18 @@
-from nautobot.extras.jobs import JobHookReceiver
+from nautobot.extras.jobs import JobHookReceiver, StringVar
 from nautobot.dcim.models import Site
 
 
 class TestJobHookReceiverLog(JobHookReceiver):
     def receive_job_hook(self, change, action, changed_object):
+        self.log_info(f"change: {change}")
+        self.log_info(f"action: {action}")
         self.log_success(changed_object.name)
 
 
 class TestJobHookReceiverChange(JobHookReceiver):
+    a_testvar_b_first = StringVar()
+    a_testvar_a_second = StringVar()
```

Comment:
What are these for?

Reply:
They were for field order testing. I moved this to a regular job